custom_components/electrasmart/climate.py

```diff
@@ -39,13 +40,23 @@
     HVAC_MODE_HEAT_COOL,
     SUPPORT_TARGET_TEMPERATURE,
     SUPPORT_FAN_MODE,
+    SUPPORT_PRESET_MODE,
     FAN_OFF,
     FAN_AUTO,
     FAN_LOW,
     FAN_MEDIUM,
     FAN_HIGH,
+    PRESET_NONE,
+    PRESET_SLEEP,
 )
 
+PRESET_SHABAT = "Shabat"
+PRESET_IFEEL = "IFeel"
+PRESET_SHABAT_SLEEP = "Shabat, Sleep"
+PRESET_SHABAT_IFEEL = "Shabat, IFeel"
+PRESET_SLEEP_IFEEL = "Sleep, IFeel"
+PRESET_SHABAT_SLEEP_IFEEL = "Shabat, Sleep, IFeel"
+
```

Comment:
Discovered by "mistake" that Sleep and Shabat can coexist, so tried to mix all up
Did not look if the interface allows for multiple presets